stream/src/ngx_stream_lua_kong_module.h

```diff
@@ -5,10 +5,12 @@
 #include <ngx_config.h>
 #include <ngx_core.h>
 #include <ngx_stream.h>
-
+#include <ngx_stream_lua_api.h>
+#include "../../src/ssl/ngx_lua_kong_ssl.h"
 
 typedef struct {
-    ngx_uint_t              proxy_ssl_disable; /* unsigned proxy_ssl_disable:1; */
+    ngx_lua_kong_ssl_ctx_t  *ssl_ctx;
```

Comment:
as `ssl_ctx` always have same lifecycle as `ngx_stream_lua_kong_ctx_t`, shall we just use the struct instead of pointer as the member? then you only
need to palloc once and free once (https://github.com/Kong/lua-kong-nginx-module/pull/50/files?diff=split&w=1#diff-9b07aee9a137ad53b90b06f49cb9559b5d484c68eabc9bff56b0732bf47cec5cR106)